package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "broccoli-lint-eslint",
-  "version": "4.2.1",
+  "version": "5.0.0",
```

Comment:
this should be done using `yarn version` to make sure that the commit is tagged correctly

Reply:
Done.